ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -73,7 +73,6 @@ endif()
 
 install(TARGETS onnx_importer EXPORT ngraphTargets
         RUNTIME DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
-        ARCHIVE DESTINATION ${NGRAPH_INSTALL_LIB} COMPONENT ngraph
```

Comment:
I suppose ONNX runtime uses `onnx_importer` directly, is not it?

Reply:
I will ask the team, I recommended to use `Core::ReadNetwork`

Reply:
No, they use:
```
    ng_function = ngraph::onnx_import::import_onnx_model(model_stream);
```